From 12fe580cbca5cf1c6fa0369ecdf757d6ac3527f8 Mon Sep 17 00:00:00 2001
From: xzhang <xzhang@xzdt>
Date: Mon, 5 Oct 2015 14:26:50 -0700
Subject: [PATCH 270/275] Revert "HIVE-11835: Type decimal(1,1) reads 0.0,
 0.00, etc from text file as NULL (Reviewed by
 Szehon)"

This reverts commit 1861781276e3134566b1bef81f883158f2e7c709.
---
 data/files/decimal_1_1.txt |   30 ------------------------------
 1 file changed, 30 deletions(-)
 delete mode 100644 data/files/decimal_1_1.txt

diff --git a/data/files/decimal_1_1.txt b/data/files/decimal_1_1.txt
deleted file mode 100644
index ec16804..0000000
--- a/data/files/decimal_1_1.txt
+++ /dev/null
@@ -1,30 +0,0 @@
-0.0
-0.0000
-.0
-0.1
-0.15
-0.9
-0.94
-0.99
-0.345
-1.0
-1
-0
-00
-22
-1E-9
--0.0
--0.0000
--.0
--0.1
--0.15
--0.9
--0.94
--0.99
--0.345
--1.0
--1
--0
--00
--22
--1E-9
-- 
1.7.9.5

